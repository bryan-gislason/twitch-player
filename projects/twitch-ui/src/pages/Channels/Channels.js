import { useTranslation } from "@twitch-player/i18n";
import React, { memo } from "react";
import AllChannels from "../../components/AllChannels";

const Channels = () => {
  const { t } = useTranslation();

  return <AllChannels heading={t("pages.channels.mainHeading")} />;
};

export default memo(Channels);
