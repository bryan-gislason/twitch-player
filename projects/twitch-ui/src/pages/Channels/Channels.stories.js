import { storiesOf } from "@storybook/react";
import { createClient } from "@twitch-player/data";
import React from "react";
import { ApolloProvider } from "react-apollo-hooks";
import Main from "../../components/Main";
import Channels from ".";

storiesOf("Pages|Channels", module).add("default", () => (
  <ApolloProvider client={createClient()}>
    <Main>
      <Channels />
    </Main>
  </ApolloProvider>
));
