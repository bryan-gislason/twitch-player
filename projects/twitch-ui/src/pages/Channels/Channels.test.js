import React from "react";
import { render } from "../../testing";
import Channels from "./Channels";

jest.mock("../../components/AllChannels", () => "mock-all-channels");

describe("Channels", () => {
  let instance;

  beforeEach(() => {
    instance = render(<Channels />);
  });

  it("renders the page", () => {
    expect(instance.asFragment()).toMatchSnapshot();
  });
});
