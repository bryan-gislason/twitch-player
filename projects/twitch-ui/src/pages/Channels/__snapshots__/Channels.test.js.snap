// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Channels renders the page 1`] = `
<DocumentFragment>
  <mock-all-channels
    heading="Live Channels"
  />
</DocumentFragment>
`;
